src/data/error.json

```diff
@@ -263,6 +263,12 @@
       "title":"error: RPC failed; curl transfer closed with outstanding read data remaining",
       "description":"typically occurs when Git is unable to complete a network request, often due to a slow or unstable connection or Small HTTPS Buffer on Client machine",
       "solutions":" increasing the Git buffer size and using a different network connection are two possible solutions that may help resolve the issue < git config http.postBuffer 524288000 < git config --global http.maxRequestBuffer 524288000 < git config --global core.compression 0"
+    },
+     {
+      "type": "push",
+      "title": "Your branch is ahead of ",
+      "description": "Some quick example text to build on the card title and make up the bulk of the card's content.error: failed to push some refs to 'git@github.com:/reponame.git' To prevent you from losing history, non-fast-forward updates were rejected Merge the remote changes (e.g. 'git pull') before pushing again. See the 'Note about fast-forwards' section of 'git push --help' for details.",
```

Comment:
Hi @aAYUSHkujur ,
Thank you for the PR, But is this the solution for the error that I mentioned? 

and you didn't add the title of the error,  you just copy paste the template that I showed.

Reply:
Sorry for the mistake. I have changed the commit.